api/src/test/java/de/cyface/api/v2/AuthorizationTest.java

```diff
@@ -30,7 +30,12 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import de.cyface.api.AuthenticatedEndpointConfig;
+import de.cyface.api.DatabaseConstants;
+import de.cyface.api.EndpointConfig;
+import de.cyface.api.PauseAndResumeAfterBodyParsing;
 import org.apache.commons.lang3.Validate;
+import org.hamcrest.MatcherAssert;
```

Comment:
assertThat from here should not be used anymore. Please use the assertThat method from JUnit directly.